scripts/gear.sh

```diff
@@ -308,7 +308,7 @@ case "$COMMAND" in
 
       rtest)
         header "Running node runtime testsuite"
-        rtest "$ROOT_DIR" "$@"; ;;
+        rtest "$ROOT_DIR" "$TARGET_DIR" "$@"; ;;
```

Comment:
will be outdated once rebased master

Reply:
I'm afraid no since `TARGET_DIR` is required. Explained to `test.sh` above